.gitattributes

```diff
@@ -0,0 +1 @@
+ziti/_version_src.py export-subst
```

Comment:
Neat. I wasn't aware you could do this. I think the goal is to keep track of the precise Git commit used to build a Git archive by interpolating tokens inside the particular file for which export-subst is enabled.